i18n/translations/fr/form-builder.json

```diff
@@ -1,4 +1,10 @@
 {
+  "element": {
+    "copy": {
+      "text": "Copy element",
```

Comment:
@anikbrazeau 

Added a couple of string here ... 

For now added this as an cds admin only button --- see PR desc